invenio/templates/deployments/web.yaml

```diff
@@ -131,6 +139,11 @@ spec:
         - mountPath: /opt/invenio/var/instance/data
           name: shared-volume
         {{- end }}
+        {{- range .Values.web.volumeMounts_extra }}
```

Comment:
shall we align naming, e.g. `extra` always as prefix or postfix and always singular or plural?